gatewaycmd/reload/reload.go

```diff
@@ -0,0 +1,76 @@
+package reload
+
+import (
+	"context"
+	"errors"
+	"github.com/TykTechnologies/gateway-sdk/pkg/apim"
```

Comment:
I would suggest to have three separete groups for imports:

```
import (
        "context"
	"errors"
	"net/http"
	
	"github.com/spf13/cobra"
	"github.com/spf13/pflag"
	
	"github.com/TykTechnologies/gateway-sdk/pkg/apim"
	"github.com/TykTechnologies/tykctl/gatewaycmd/shared"
	"github.com/TykTechnologies/tykctl/internal"
)
```

Reply:
I have added    
`  goimports:
    local-prefixes: "github.com/TykTechnologies"` in the  .golangci.yml to help with this.
:https://github.com/TykTechnologies/tykctl/blob/e6e1846b29dad7760ce248677db3dc3391f08ed3/.golangci.yml#L103